packages/bpk-component-popover/README.md

```diff
@@ -84,6 +85,7 @@ class App extends Component {
 | portalClassName       | string                                    | false    | null          |
 | renderTarget          | func                                      | false    | null          |
 | popperModifiers       | object                                    | false    | null          |
+| tabIndex              | string                                    | false    | "1"           |
```

Comment:
This should have type `number`

Reply:
Also I think this name could be interpreted as the tabIndex for the popover itself. Could we rename it to something like `closeButtonTabIndex`?

Reply:
I actually think we should avoid cherry picking specific props and instead add a `closeButtonProps` prop to allow the consumer to spread anything on that button. use case: adding an event for analytics purposes...